src/main/java/engine/drivers/WebDriverFactory.java

```diff
@@ -0,0 +1,14 @@
+package engine.drivers;
+
+import enums.BrowserName;
+import org.openqa.selenium.WebDriver;
+
+public class WebDriverFactory {
+    public static WebDriver getWebDriver(BrowserName browserName) {
+        return switch (browserName) {
+            case CHROME -> new ChromeDriverManager().getDriver();
+            case FIREFOX -> new FirefoxDriverManager().getDriver();
+            default -> throw new UnsupportedOperationException("For now the only supported browsers are Google Chrome and Mozilla Firefox.");
```

Comment:
I would make this message bases on browserName parameter and all values from BrowserName enum - so we will not need to change this message each time we add new browser:)